Software/Development/SemesterProject/App/analogSignal.c

```diff
@@ -0,0 +1,98 @@
+//###########################################################################
+//FILE: signal.c
+//
+//Project: Electric Go-kart
+//
+//Author: Nicolai, 6/3/19
+//
+//Notes:    Signal. This will create the signals for each ADC.
+//
+//
+//###########################################################################
+
+//###########################################################################
+
+//
+// Includes
+//
+
+#include "DSP28x_Project.h"     // Device Headerfile and Examples Include File
+#include "AnalogSignal.h"
+#include <math.h>

```

Comment:
maybe adding all includes to analogSignal.h?

Reply:
You are right